test/wallet-rpc-test.js

```diff
@@ -466,6 +466,138 @@ describe('Wallet RPC Methods', function() {
     });
   });
 
+  describe('signmessagewithname & verifymessagewithname (reveal in progress)', () => {
```

Comment:
So I think these tests can the REVOKE tests can be combined into the existing `describe` block for simplicity. Change line 401 to mine 1 block only, then the first few `it` blocks can be the not-closed-yet tests. Then after line 466 you can mine more blocks and squeeze in the REVOKE test, so that `describe` block is all one long timeline.